pkg/agent/endpoints/workload/handler.go

```diff
@@ -117,7 +116,14 @@ func (h *Handler) FetchJWTABundles(req *workload.JWTABundlesRequest, stream work
 	return nil
 }
 
-func (h *Handler) ValidateJWTASVID(ctx context.Context, req *workload.ValidateJWTASVIDRequest) (*workload.ValidateJWTASVIDResponse, error) {
+func (h *Handler) ValidateJWTSVID(ctx context.Context, req *workload.ValidateJWTSVIDRequest) (*workload.ValidateJWTSVIDResponse, error) {
+	if req.Audience == "" {
+		return nil, errs.New("audience must be specified")
+	}
+	if req.Svid == "" {
+		return nil, errs.New("svid must be specified")
```

Comment:
seems like these should also be InvalidArgument gRPC code?

Reply:
👍